Add a callout for chat on US holiday (#21791)
DataDog/documentation: content/en/getting_started/support/_index.md

  text: "Agent Troubleshooting"
---

> {{< callout url="https://docs.datadoghq.com/account_management/guide/access-your-support-ticket/" >}}

Reviewer: ```suggestion
{{< callout header="false" >}}
```

> Please note that our US offices will be closed on Monday, February 19th in honor of President’s Day. We will be operating with limited staffing, and as a result, Live Chat will not be available on President’s Day for Standard Support customers during normal US business hours. Premier Support customers will retain access to chat 24x7x365. For any urgent issues, please contact us through our Support Portal.

Reviewer: ```suggestion
Our US offices are closed on Monday, February 19th for Presidents' Day. Support will be operating with limited staffing, and so Live Chat will not be available on Presidents' Day for Standard Support customers during normal US business hours. Premier Support customers retain access to chat 24x7x365. For urgent issues, contact us through the <a href="https://docs.datadoghq.com/account_management/guide/access-your-support-ticket/">Support Portal</a>.
```
